doc/README_linux.md

```diff
@@ -50,4 +50,9 @@ slots:
     name: `APPLICATION_ID`
 ```
 - You can refer to these two repositories for more details: [FlatHub setup](https://github.com/flathub/io.appflowy.AppFlowy) and [Snapcraft setup](https://github.com/LucasXu0/appflowy-snap/blob/main/snap/snapcraft.yaml).
-
+- If you created the .deb or .rpm installer with [Flutter Distributor](https://pub.dev/packages/flutter_distributor), please ensure that in the make_config.yaml you set
+``` yaml
+supported_mime_type:
+  - x-scheme-handler/my_custom_scheme # necessary so that the flutter app can open custom urls of type my_custom_scheme:/...
+  - x-scheme-handler/https # this allows to open https scheme (further setup is needed to open https links by default with your app)
```

Comment:
I'd prefer to remove https as it will likely open un-intended URLs with this kind of setup.

Reply:
sounds good.